frontend/src/metabase/home/homepage/components/XraySection/XraySection.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useMemo } from "react";
+import _ from "underscore";
+import { t } from "ttag";
+import * as Urls from "metabase/lib/urls";
+import { Database, DatabaseCandidate } from "metabase-types/api";
+import {
+  DatabaseIcon,
+  DatabaseLink,
+  DatabaseTitle,
+  SectionTitle,
+  XrayCard,
+  XrayIcon,
+  XrayIconContainer,
+  XrayList,
+  XrayTitle,
+  XrayTitlePrimary,
+  XrayTitleSecondary,
+} from "./XraySection.styled";
+
+export interface XraySectionProps {
+  database?: Database;
+  databaseCandidates: DatabaseCandidate[];
+}
+
+const XraySection = ({
+  database,
+  databaseCandidates,
+}: XraySectionProps): JSX.Element => {
+  const isSample = !database || database.is_sample;
+  const tables = databaseCandidates.flatMap(d => d.tables);
+  const tableCount = tables.length;
+  const tableMessages = useMemo(() => getMessages(tableCount), [tableCount]);
+
+  return (
+    <div>
+      {isSample ? (
+        <SectionTitle>
+          {t`Try out these sample x-rays to see what Metabase can do.`}
+        </SectionTitle>
+      ) : (
+        <SectionTitle>
+          {t`Here are some explorations of`}
```

Comment:
I don't have a great alternative yet but I think we might want to give a bit more context for what the heck these are and more actively frame it as a quick way to assist you getting started. Maybe "Get started quickly with some pre-built explorations of your {name} data."?